src/components/Panel.tsx

```diff
@@ -105,17 +113,18 @@ const Panel: React.FC<PanelProps> = (props) => {
   function updateState(data: Pane, forceOpen = false) {
     const { minSize } = props
     const { hidden } = state
-
     const size = Math.max(minSize || 0, data.size)
-
     const collapsed = !forceOpen && (data.collapsed || hidden)
-
     setState((state) => ({
       ...state,
       originalSize: size,
       size: collapsed ? 0 : size,
       collapsed,
     }))
+
+    if (forceOpen === false && data.collapsed === true) {
+      dispatch({ selectedComment: undefined })
+    }
```

Comment:
it should work without this change, can you please remove it and check if it's working

Reply:
It does not work without this...